app/src/main/res/drawable-ldrtl/bottom_left_rounded_rect_coming_soon_background.xml

```diff
@@ -7,7 +7,7 @@
     android:topRightRadius="0dp" />
   <gradient
     android:angle="360"
-    android:endColor="@color/coming_soon_end"
-    android:startColor="@color/coming_soon_start"
+    android:endColor="@color/color_def_grape_violet"
```

Comment:
Do not use color from `color_defs.xml` directly here. We should only use colors from `component_colors.xml` file.

Reply:
@rt4914 like in this only one color is used and its also not changed in the dark mode so can I use this color or make a new value in the component colors?

Reply:
You will need to create new value in components for such case too.